automerge-cli/Cargo.toml

```diff
@@ -9,7 +9,8 @@ name = "automerge"
 path = "src/main.rs"
 
 [dependencies]
-clap = { git = "https://github.com/clap-rs/clap/" }
+structopt = "0.3"
```

Comment:
This is interesting, I was under the impression that structopt is now part of clap as part of clap v3, which is what is suggested by this crate https://github.com/clap-rs/clap/tree/master/clap_derive

I need to do more research but it feels like the ecosystem is consolidating around clap, thoughts?

Reply:
Oh, fascinating! I've been using `structopt` for a while now and just kinda ignored what was happening over at `clap`. I'm all for using `clap`@3, I'll fix this up.

Reply:
⚡ https://github.com/automerge/automerge-rs/pull/17/commits/33906bd15957e2e2e60d4a23ff86619670d2e238